src/core/RoundedBox.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react'
 import { Mesh, Shape, ExtrudeGeometry } from 'three'
 import { NamedArrayTuple } from '../helpers/ts-utils'
-import { toCreasedNormals } from 'three-stdlib/utils/BufferGeometryUtils'
```

Comment:
@CodyJasonBennett I did this :| but can you explain to me why this was a bad idea? (so I won't make the same error next time) thx

Reply:
Drei doesn't transpile subpaths AFAIK. We can fix this with `package#exports` there but that would be a breaking change. For now, we just import from the root.